pricetracker/src/components/layout/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import propTypes from 'prop-types'
+
+
+const Navbar = ({ icon, title }) => {
+
+
+
+    return (
+        <nav className='navbar bg-primary'>
+            <h1>
+                <i className={icon} /> {title}
+            </h1>
+            <ul>
+                <li>Home</li>
+                <li>About us</li>
+            </ul>
+        </nav>
+    );
+}
+
+Navbar.defaultProps = {
+    title: "Price Tracker",
+    icon: "fas fa-hand-holding-usd"
+}
+Navbar.propTypes = {
+    title: propTypes.string.isRequired,
+    icon: propTypes.string.isRequired
+}
```

Comment:
If your project would have more similar type checks, you can maybe check out Typescript. :D
"Typescript validates types at compile time, whereas PropTypes are checked at runtime. Typescript is useful when you are writing code: it will warn you if you pass an argument of the wrong type to your React components, give you autocomplete for function calls, etc."

Reply:
thanks @AnandBaburajan  for your suggestions . i will definitly  follow your suggestions 💯 👍